reana/reana_dev/utils.py

```diff
@@ -724,3 +724,24 @@ def validate_mode_option(ctx, param, value):
             "Supported values are '{}'.".format("', '".join(CLUSTER_DEPLOYMENT_MODES))
         )
     return value
+
+
+def get_current_tag(component):
```

Comment:
What do you think of the following possible names? 
- `get_docker_tag`
    1. we assume docker tags will always be in semver2 format
    2. goes in harmony with `get_git_tag `
- `get_component_semver2`
    1. does exactly what it says

Reply:
Nice suggestion, I changed the code to use `get_docker_tag`. If anyone has any other opinions, let me know